api/src/services/healthService.js

```diff
@@ -94,6 +96,33 @@ function splitParams(params) {
   return params && params.split(SEPARATOR)
 }
 
+async function liveness(req, res) {
+  res.status(200).json({
+    msg: 'Liveness probe is ok',
+  });
+}
+
+async function readiness(req, res) {
+  let errors = [];
+  
+  try {
+    await getClient();
+  } catch (err) {
+    errors.push(`Unable to connect to database:${err.message}`);
+  }
+
+  const accounts = cacheService.getAccounts();
```

Comment:
Introduce a new method in `cacheService` with name `isReady` and incapsulate the logic there.

Reply:
fixed